sources/packages/backend/apps/api/src/services/announcements/announcements.service.ts

```diff
@@ -13,16 +13,13 @@ export class AnnouncementsService extends RecordDataModelService<Announcements>
    * @returns system announcements list.
    */
   async getAnnouncements(): Promise<Announcements[]> {
+    const now = new Date();
     return this.repo.find({
-      select: {
-        id: true,
-        messageTitle: true,
-        message: true,
-        target: true,
-        startDate: true,
-        endDate: true,
-      },
       order: { startDate: "ASC" },
+      where: {
```

Comment:
Minor: We follow the following order: `select` , `where` , `order`. Please put `where` before `order`.

Reply:
updated